engine-tests/src/main/java/org/terasology/TerasologyTestingEnvironment.java

```diff
@@ -104,12 +104,12 @@ public void setup() throws Exception {
         EntitySystemSetupUtil.addReflectionBasedLibraries(context);
         EntitySystemSetupUtil.addEntityManagementRelatedClasses(context);
         engineEntityManager = context.get(EngineEntityManager.class);
-        BlockManager blockManager = context.get(BlockManager.class);
+        BlockManager blockManagerLocal = context.get(BlockManager.class); // added Local to avoid HiddenFieldCheck
```

Comment:
This was an instance of `HiddenFieldCheck`, with CheckStyle note `'blockManager' hides a field.`